earth_enterprise/src/fusion/rasterfuse/TmeshPackgenTraverser.cpp

```diff
@@ -150,6 +154,15 @@ TmeshPackgenTraverser::TmeshPackgenTraverser(
       writer_(file_pool_, output, true /* overwrite */),
       extra_cache_(20000 /* apx 160M */),
       not_covered_tiles_exist_(false) {
+  if(decimation_threshold == 0.0)
+  {
+    decimate_ = false;
```

Comment:
I think we should set `decimate_ = config.decimate ` to maintain the same behavior as previous versions.  That does result in the odd situation where `decimate_` could be false while `decimation_threshold` is non-zero or vice versa, but I'm OK with that at this point.  We may want to do some additional research into what the decimate_ flag is used for, but that would be a task for a different ticket.